src/main/java/uk/gov/hmcts/reform/civil/helpers/BundleRequestMapper.java

```diff
@@ -185,88 +186,101 @@ private List<Element<BundlingRequestDocument>> mapSystemGeneratedcaseDocument(Li
-                                                        .documentType(DocumentType.SDO_ORDER.name()).build());
-            }
+        }
+        if (null != orderSDODocumentDJ) {
+            bundlingSystemGeneratedCaseDocs.add(BundlingRequestDocument.builder()
+                                                    .documentFileName(orderSDODocumentDJ.getDocumentFileName())
+                                                    .documentLink(DocumentLink.builder()
+                                                                      .documentUrl(orderSDODocumentDJ.getDocumentUrl())
+                                                                      .documentBinaryUrl(orderSDODocumentDJ.getDocumentBinaryUrl())
+                                                                      .documentFilename(orderSDODocumentDJ.getDocumentFileName()).build())
+                                                    .documentType(DocumentType.SDO_ORDER.name()).build());
         }
         return ElementUtils.wrapElements(bundlingSystemGeneratedCaseDocs);
     }
 
     private List<Element<BundlingRequestDocument>> mapUploadEvidenceWitnessDoc(List<Element<UploadEvidenceWitness>> uploadEvidenceWitness, String displayName) {
         List<BundlingRequestDocument> bundlingWitnessDocs = new ArrayList<>();
-        if (!Optional.ofNullable(uploadEvidenceWitness).isEmpty()) {
-            uploadEvidenceWitness.forEach(witnessDocs -> {
-                StringBuilder fileNameBuilder = new StringBuilder();
-                fileNameBuilder.append(displayName);
-                if (Optional.ofNullable(witnessDocs.getValue().getWitnessOptionName()).isPresent()) {
-                    fileNameBuilder.append("_" + witnessDocs.getValue().getWitnessOptionName());
-                }
-                if (Optional.ofNullable(witnessDocs.getValue().getWitnessOptionUploadDate()).isPresent()) {
-                    fileNameBuilder.append("_" + DateFormatHelper.formatLocalDate(witnessDocs.getValue()
-                                                                                      .getWitnessOptionUploadDate(), "ddMMyyyy"));
-                }
-                Document document = witnessDocs.getValue().getWitnessOptionDocument();
-                bundlingWitnessDocs.add(BundlingRequestDocument.builder()
-                                            .documentFileName(fileNameBuilder.toString())
-                                            .documentLink(DocumentLink.builder()
-                                                              .documentUrl(document.getDocumentUrl())
-                                                              .documentBinaryUrl(document.getDocumentBinaryUrl())
-                                                              .documentFilename(document.getDocumentFileName()).build())
-                                            .build());
-
-            });
+        if (null == uploadEvidenceWitness || uploadEvidenceWitness.isEmpty()) {
```

Comment:
You don't need the `isEmpty()` check. If the collection is empty, no iteration will be ran.

Replace it as follows:
```suggestion
        if (null == uploadEvidenceWitness) {
```